jsonschema/_utils.py

```diff
@@ -38,6 +38,22 @@ def __repr__(self):
         return repr(self.store)
 
 
+class Cache(object):
```

Comment:
Cool, the interface is identical, I'll only install `repoze.lru` for python2.

Adding a dependency required a small change to where `__version__` is defined.  

When trying to run setup, `setup.py` imports `__init__.py` imports `validators.py`  imports `comat.py` tries to import `repoze.lru` which will fail before the virtualenv doesn't exist yet.  So I created a `version.py`, moved `__version__` there, imported it from both `setup.py` and `__init__.py` and use `exec()` to read it without importing it.  

I believe this is a pretty common way of handling this problem.